simple_app/utils.py

```diff
@@ -8,12 +8,20 @@
 from markdown2 import markdown  # using markdown formatting
 import numpy as np  # numerical python
 import pandas as pd  # running dataframes
+from tqdm import tqdm
 from wtforms import StringField, SubmitField  # web forms
 # internal packages
 import argparse  # system arguments
 from typing import Union, List  # type hinting
 
 
+PHOTOMETRIC_FILTERS = {'WISE.W1': 33526.00, 'WISE.W2': 46028.00, 'WISE.W3': 115608.00, 'WISE.W4': 220883.00,
+                       '2MASS.J': 12350.00, '2MASS.H': 16620.00, '2MASS.Ks': 21590.00,
+                       'GAIA2.G': 6230.00, 'GAIA2.Grp': 7730.00,
+                       'GAIA3.G': 6217.59, 'GAIA3.Grp': 7769.02,
+                       'IRAC.I1': 35378.41, 'IRAC.I2': 44780.49, 'IRAC.I3': 56961.77, 'IRAC.I4': 77978.39}
```

Comment:
controversial part was the creation of a dictionary with hardcoded reference values for central wavelengths (from VOSA filters), for all of our current photometric bands -- needed to do so, so the code only makes colours which are blue-red

Reply:
I think this data is in the Photometry Filters table. Any reason not to pull from there?

Reply:
Ah yep thankyou! Knew we'd set up something like that, c.f. new commits